novuRajan/gannt-chart
# Sidebar removed

## Description of the problem this PR solves
SideBar should be hide according to the input given by user
## Details of how the solution has been implemented
Svg was place inside the div-date. The CSS of the SVG was removed where left:21 rem was placed.
## Testing Instructions
inside index.html

```
 GanttChart.createGanttChart(data,{
        containerId: 'chart',
        sidebar: false,
    });
```

npm run dev is required 

you will see the sidebar hidden and only date and SVG are shown 
when sidebar is made true you will be able to see the sidebar

## Browsers tested
Select all browsers the solution was tested on <br>
*(test on as many as possible if the solution is front-end, otherwise this section is optional)*

## Notes
Any further information required for this PR to be reviewed and tested
